statik/database.py

```diff
@@ -418,6 +418,14 @@ def __init__(self, model=None, session=None, **kwargs):
                     "Attempting to look up primary keys for ManyToMany " +
                     "field relationship: %s", self.field_values[field_name]
                 )
+
+                # check if non-string items are present
+                if not all(isinstance(item, str) for item in self.field_values[field_name]):
+                    logger.warning("Non-string values in array (field: %s, model: %s): %s",
```

Comment:
on https://github.com/coala/coAST/commit/e0b4e0134b1243722721c65b907dae12918ff17a , this spits out

```console
Non-string values in array (field: keywords, model: Language): ['access', 'addressof', 'alias', 'and', 'attribute', 'appactivate', 'append', 'as', 'beep', 'begin', 'binary', 'byval', 'byref', 'call', 'case', 'class', 'close', 'chdir', 'chdrive', 'collection', 'const', 'declare', 'defbool', 'defbyte', 'defcur', 'defdbl', 'defdate', 'defdec', 'defint', 'deflng', 'defobj', 'defsng', 'defstr', 'defvar', 'deletesetting', 'dim', 'do', 'each', 'else', 'elseif', 'end', 'enum', 'eqv', 'erase', 'error', 'event', 'filecopy', 'friend', 'for', 'function', 'get', 'global', 'gosub', 'goto', 'if', 'imp', 'implements', 'in', 'input', 'is', 'kill', 'load', 'lock', 'loop', 'len', 'let', 'lib', 'like', 'lset', 'me', 'mid', 'mkdir', 'mod', 'name', 'next', 'new', 'not', 'nothing', True, 'open', 'optional', 'or', 'output', 'paramarray', 'preserve', 'print', 'private', 'public', 'put', 'random', 'randomize', 'raiseevent', 'read', 'redim', 'rem', 'reset', 'resume', 'return', 'rmdir', 'rset', 'savepicture', 'savesetting', 'seek', 'select', 'sendkeys', 'set', 'setattr', 'shared', 'spc', 'static', 'step', 'stop', 'sub', 'tab', 'text', 'then', 'to', 'type', 'typeof', 'unload', 'unlock', 'until', 'variant', 'version', 'wend', 'while', 'width', 'with', 'withevents', 'write', 'xor']
...
Non-string values in array (field: keywords, model: Language): ['and', 'array', 'asm', 'begin', 'break', 'case', 'const', 'constructor', 'continue', 'destructor', 'div', 'do', 'downto', 'else', 'end', 'file', 'for', 'function', 'goto', 'if', 'implementation', 'in', 'inline', 'interface', 'label', 'mod', 'nil', 'not', 'object', 'of', True, 'operator', 'or', 'packed', 'procedure', 'program', 'record', 'repeat', 'set', 'shl', 'shr', 'then', 'to', 'type', 'unit', 'until', 'uses', 'var', 'while', 'with', 'xor']
```

I cant see which file this problem occurred in, and I cant (easily) see which value was not a string.

The warning isnt precise enough to be helpful.  The first contains a `True` , and the second contains a `True`.  But it is hard to find them in the warning.  And then, I still dont know which file the problem is.

More precise warnings please ;-)